SSA/Projects/RISCV64/Semantics.lean

```diff
@@ -25,22 +32,86 @@ def UTYPE_pure64_AUIPC (imm : BitVec 20) (pc : BitVec 64) : BitVec 64 :=
   BitVec.add (BitVec.signExtend 64 (BitVec.append imm (0x000 : BitVec 12))) pc
 
 def SHIFTIWOP_pure64_RISCV_SLLIW (shamt : BitVec 5) (rs1_val : BitVec 64) : BitVec 64 :=
-  BitVec.signExtend 64 ((BitVec.extractLsb' 0 32 rs1_val) <<< shamt)
+    BitVec.signExtend 64 (BitVec.shiftLeft (BitVec.extractLsb' 0 32 rs1_val) (shamt).toNat)
+
+def SHIFTIWOP_pure64_RISCV_SLLIW_bv (shamt : BitVec 5) (rs1_val : BitVec 64) : BitVec 64 :=
+    BitVec.signExtend 64 ((BitVec.extractLsb' 0 32 rs1_val)  <<< shamt)
```

Comment:
Your new definitions use an indentation of four whitespace vs. two in the original code. Lean uses two whitespace, hence I would suggest to stick to two whitespace as well (throughout the file).